server/auth/types/saml/routes.ts

```diff
@@ -194,7 +194,8 @@ export class SamlAuthRoutes {
           const authInfo = await this.securityClient.authinfo(request);
           this.sessionStorageFactory.asScoped(request).clear();
           // TODO: need a default logout page
-          const redirectUrl = authInfo.sso_logout_url || '/';
+          const redirectUrl =
+            authInfo.sso_logout_url || `${this.coreSetup.http.basePath.serverBasePath || '/'}`;
```

Comment:
Could it be simplified to `authInfo.sso_logout_url || this.coreSetup.http.basePath.serverBasePath || '/'`?

Reply:
updated